### PR TITLE
Asset server is disabled in production

### PR DESCRIPTION
**Pull request by aaronjensen**

### CODE REVIEW

**Comment by aaronjensen** [comment]
@rafaelfranca is this what you had in mind?

**Comment by rafaelfranca** [comment]
Yes! Thank you

**Comment by aaronjensen** [comment]
Great, thank you.

File: lib/propshaft/railtie.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ class Railtie < ::Rails::Railtie
       [ "text/javascript", Propshaft::Compilers::SourceMappingUrls ]
     ]
     config.assets.sweep_cache = Rails.env.development?
+    config.assets.server = !Rails.env.production?
```
Comment (dhh) [inline lib/propshaft/railtie.rb:15]: Think we need the opposite here. Turning the server on in dev + test, so it'll remain off in other envs like staging and beta.

Comment (aaronjensen) [reply lib/propshaft/railtie.rb:15]: https://github.com/rails/propshaft/pull/62

**Pull request merged**